monsteer/CMakeLists.txt

```diff
@@ -18,5 +18,5 @@ set(MONSTEER_LINK_LIBRARIES PUBLIC Brion Lunchbox zeq)
 
 common_library(Monsteer)
 
+add_subdirectory(brionplugin)
```

Comment:
I had started with the shorter version, but the side effect is that it creates a conflict between monsteer::brion and brion which forces you to do ::brion everywhere. So I settled for this which I find more clear and elegant. But up to you; for me, we could forget about the extra namespace altogether ;-)